### PR TITLE
PXB-2142 transition key information is not masked

### PR DESCRIPTION
**Pull request by rahulmalik87**
PXB-2142 transition key information is not masked

### CODE REVIEW

**Comment by satya-bodapati** [comment]
> _No description provided._

Please provide description message

File: storage/innobase/xtrabackup/src/backup_mysql.cc [hunk 1]
```diff
@@ -2096,6 +2096,14 @@ char *make_argv(char *buf, size_t len, int argc, char **argv)
 				strlen("--encrypt_key")) == 0) {
 			arg = "--encrypt_key=...";
 		}
+		if (strncmp(*argv, "--transition-key",
+				strlen("--transition-key")) == 0) {
```
Comment (satya-bodapati) [inline storage/innobase/xtrabackup/src/backup_mysql.cc:2100]: strlen() should ideally be aligned to *argv.. but it looks like old code is doing wrong alignment too..

So either fix all strlen alignment or leave it as it is.. 
choice is yours :)

**Review by satya-bodapati — APPROVED** [review]
Please fix description in github page

File: storage/innobase/xtrabackup/src/backup_mysql.cc (hunk 1 above)
Comment (rahulmalik87) [reply storage/innobase/xtrabackup/src/backup_mysql.cc:2100]: I will leave it as it for now.

**Pull request merged**